### PR TITLE
Update Object Model & fix minor discrepancies

### PR DESCRIPTION
**Pull request by kterusaki**

### CODE REVIEW

File: source/includes/object-model/_overview.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
```
Comment (kterusaki) [inline source/includes/object-model/_overview.md]: @goldensun8891 would love your input here

File: source/includes/object-model/_datatypes.md [hunk 2]
```diff
@@ -2,16 +2,16 @@
 
 dataType | Description
 -------- | -----------
-Contact  | This type of field stores a Contact within your SalesforceIQ Organization. The value stored is a contactID (UUID for a Contact). To edit this type of field you can pass in the contactID or the contact's email address.
+Contact  | This type of field stores a Contact within your SalesforceIQ Organization. The value stored is a contactID (Id for a Contact). To edit this type of field you can pass in the contactID or the contact's email address.
 Datetime | This type of field stores a particular Date & Time in UTC milliseconds past the epoch.
 Email    | This type of field stores an email address of a Contact. (Contact Object only)
-File     | This type of field stores a file that gets uploaded via the web. The value stored is a UUID for the file. (List Item Object only)
+File     | This type of field stores a file that gets uploaded via the web. The value stored is a IdId the file. (List Item Object only)
```
Comment (HJewkes) [inline source/includes/object-model/_datatypes.md]: Id

**Comment by HJewkes** [comment]
One minor fix where you say IdId

It would also be great to add a little description of how Accounts and Contacts relate to one another in the Object Model text section

**Comment by kterusaki** [comment]
@HJewkes Definitely, I'll make sure to get in a section about how Accounts & Contacts relate to each other.

Do you guys know when you'll be pushing the Account model changes? I took a look at the SPEC, but I wanted to play around with it just so I have a better understanding of how it works before I write up that description. I think having context on how it behaves through the UI will be useful when writing this particular section.

**Comment by HJewkes** [comment]
We have a cherry pick hopefully coming out today that will let us re-ramp the riqex on monday

**Pull request merged**